index.html

```diff
@@ -648,26 +648,31 @@ <h3 class="Text Text--spacedSmall">Jayanth Noonnety</h3>
   <div class="Container">
     <h2 class="Text Text--heading u-marginBottom"><i class="Icon Icon--arrow Icon--inline"></i>FAQ's</h2>
     <div class="faq-c">
-      <div class="faq-q"><span class="faq-t">+</span>Question to be added here</div>
+      <div class="faq-q"><span class="faq-t">+</span>What is a hackathon?</div>
       <div class="faq-a">
-        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
+        <p>A hackathon is a design sprint-like event in which computer programmers and others involved in software development, including graphic designers, interface designers, project managers, and others, often including subject-matter-experts, collaborate intensively on software projects.</p>
       </div>
     </div>
 
     <div class="faq-c">
-      <div class="faq-q"><span class="faq-t">+</span>Question to be added here</div>
+      <div class="faq-q"><span class="faq-t">+</span>Is there any eligibility criteria for IIITA Hacks?</div>
       <div class="faq-a">
-        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
+        <p>If you are currently a student of IIIT-A or IIIT-L or RGIIT you can apply for this event, mentioning your enrollment number.</p>
```

Comment:
This should be changed as we know MNNIT people are also there.

Reply:
Alright , on it.